### ISSUE TITLE
Cannot open loom file

### ISSUE THREAD

**Issue by yonatans2** [mojaveazure/loomR#68]
The package fails to open the file [dev_all.agg.loom](https://storage.googleapis.com/linnarsson-lab-loom/dev_all.agg.loom) (from Mouse Brain Atlas), giving the following error:
> clust_mat = connect(filename='dev_all.agg.loom')
Error in validateLoom(object = self) :
  There can only be 5 groups in the loom file: 'row_attrs', 'col_attrs', 'layers', 'row_graphs', 'col_graphs'

Would appreciate any help.

Thanks,
Yonatan

**Comment by squatrim** [comment]
I have been having the same error with the same file.

My session info can be  found below.

Thanks
Best,
Massimo

R version 4.0.2 (2020-06-22)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Mojave 10.14.6

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] loomR_0.2.1.9000 hdf5r_1.3.2      R6_2.5.0        

loaded via a namespace (and not attached):
 [1] compiler_4.0.2    prettyunits_1.1.1 remotes_2.2.0     tools_4.0.2       testthat_2.3.2    digest_0.6.25    
 [7] pkgbuild_1.1.0    pkgload_1.1.0     bit_4.0.4         lattice_0.20-41   memoise_1.1.0     rlang_0.4.7      
[13] Matrix_1.2-18     cli_2.0.2         rstudioapi_0.11   parallel_4.0.2    curl_4.3          xfun_0.16        
[19] stringr_1.4.0     withr_2.2.0       desc_1.2.0        fs_1.4.2          devtools_2.3.1    rprojroot_1.3-2  
[25] bit64_4.0.5       grid_4.0.2        glue_1.4.1        processx_3.4.3    pbapply_1.4-3     fansi_0.4.1      
[31] sessioninfo_1.1.1 callr_3.4.3       magrittr_1.5      backports_1.1.8   ps_1.3.3          ellipsis_0.3.1   
[37] usethis_1.6.1     assertthat_0.2.1  tinytex_0.25      stringi_1.5.3     crayon_1.3.4

**Comment by FloWuenne** [comment]
I was running into the same issue today and I believe this is answered in the following issue: #55

TLDR: the solution is to add the skip.validate = TRUE argument to your connect command! 

```
lfile <- connect(filename = your_loom_file,
                 mode = "r+",
                 skip.validate = TRUE)
                 ```

**Comment by squatrim** [comment]
Thanks Flo!
`skip.validate = TRUE` indeed solved the issue. I cannot still get the gene names out of the matrix, but I guess that is related to the actual file I am working with.
